web/nginx/snippets/general.conf

```diff
@@ -0,0 +1,34 @@
+# https://nginxconfig.io/
```

Comment:
I have very little idea what these files do, but I assume they are like modules that can be loaded into nginx as config files so you don't put them all in one file

Reply:
Like c++ headers (maybe a bad example), we can include those "snippets" in individual nginx config files.